create-neon/src/bin/create-neon.ts

```diff
@@ -30,7 +31,12 @@ function inferVersions(): Versions {
 }
 
 async function main(name: string) {
+  let workingDirPath: string = process.cwd()
+  let tmpDirPath: string = tmpdir()
+
   try {
+    //change PWD to temp
+    process.chdir(tmpDirPath)
     await fs.mkdir(name);
```

Comment:
In order to avoid potential collisions in the `tmpdir` can you use a random directory name?

https://nodejs.org/api/fs.html#fspromisesmkdtempprefix-options